CondCore/Utilities/scripts/conddb

```diff
@@ -739,6 +739,24 @@ def _get_last_frozen_since( session, tagName, fcsr=None ):
     logging.debug('Last frozen since in destination tag is %s' %res)
     return res
 
+def _get_maxtime_for_boost_version( session, timeType, boost_version):
+    BoostRunMap = session.get_dbtype(conddb.BoostRunMap)
+    q = session.query(BoostRunMap).order_by(BoostRunMap.boost_version.asc())
+    time = (maxSince,maxSince,maxSince)
+    for r in q:
+        r= _rawdict(r)
+        if boost_version < r['boost_version']:
+            tlumi = r['run_number']<<32|0x1
+            time = (r['run_number'],tlumi,r['run_start_time'])
+            break
+    if timeType=='Run':
+        return time[0]
+    elif timeType=='Run':
+        return time[1]
```

Comment:
Are these supposed to be both `timeType == 'Run'` ?

Reply:
yes. not relevant for the above issue (all O2Os populates either run or timestamp tags), but definitely logically wrong.